sdk/src/builder.rs

```diff
@@ -1050,7 +1050,15 @@ impl Builder {
         fragment_paths: &Vec<std::path::PathBuf>,
         output_path: P,
     ) -> Result<()> {
-        self.set_asset_from_dest(output_path.as_ref())?;
+        if !output_path.as_ref().exists() {
+            // ensure the path exists
+            std::fs::create_dir_all(output_path.as_ref())?;
+        } else {
+            // if the file exists, we need to remove it
```

Comment:
technically this should be "if the folder exists". I'm not positive that's a concern in this case, since we could still write to the folder, and I don't think we have -f/force option wired up here either.

Reply:
I didn't change the behavior because all of the -f behavior is in c2patool.  It is the case here as well.  If the asset file is found at the output location an error is returned just like other assets.  I will fix the comment.